source/agora/test/Base.d

```diff
@@ -1972,7 +1972,7 @@ public struct TestConf
 
     /// max failed requests before a node is banned
     /// Matches the eponymous field in the `banman` section.
-    public size_t max_failed_requests = 100;
+    public size_t max_failed_requests = 10_000; // default to not banning in tests
```

Comment:
Note that banning is only done on non-validator, so I doubt that's it.

Reply:
Some tests have `FullNodes` too. But it probably will not solve much.

Reply:
I saw some nodes are banned in unit tests sometimes. We should check why some nodes are banned just in test cases, which is unexpected. I'm not sure we need to create a new issue about that though.

Reply:
The nodes do ban themselves, so you will see a message saying "node-7" banned, but it will be emitted from node-7.